src/Service/Storage/S3.php

```diff
@@ -307,7 +307,7 @@ public function validateConfiguration(): bool
         }
 
         if ($this->purpose === StorageInterface::PURPOSE_ANONYMISED_DATA
-            && !$this->getConfig()->get(Option::STORAGE_DATA_BUCKET, true)
+            && !$this->getConfig()->get(Option::STORAGE_ANONYMISED_BUCKET, true)
         ) {
             throw new ConfigurationException('A data bucket needs to be defined');
```

Comment:
Might need to change this error message too, since the config variable has changed.